portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -31,6 +31,15 @@
 import com.google.appengine.api.datastore.Query;
 import com.google.appengine.api.datastore.Query.SortDirection;
 
+import java.util.Date;
+import java.time.LocalDateTime;
+import java.sql.Timestamp;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Calendar;
+import java.util.TimeZone;
+import java.util.*;
```

Comment:
That's plenty datetime related classes. Either LocalDateTime or ZonedDateTime should be enough, might be also java.time.format.DateTimeFormatter if toString() doesn't work for your case.